learning/tour-of-beam/frontend/lib/components/login/login_button.dart

```diff
@@ -39,13 +39,13 @@ class LoginButton extends StatelessWidget {
     OverlayEntry? overlay;
```

Comment:
1. Create an empty `ChangeNotifier`.
2. In `DismissibleOverlay.close` call `notifyListeners` on that notifier.
3. When the overlay is initialized, call `notifier.addListener(overlay.remove);`

To avoid warnings, we will have to subclass `ChangeNotifer` to expose its `notifyListeners` method. But first look for an existing `ChangeNotifier` subclass with a public `notifyListeners` method.